### ISSUE TITLE
ordering instructions issue - copper weight on 4+ layer boards

### ISSUE THREAD

**Issue by lewispg228** [sparkfun/SparkFun_KiCad_Panelizer#8]
Ordering instructions currently don't correctly parse inner and outer copper weights.

My ordering instructions in my panel board file look like this:

![image](https://github.com/sparkfun/SparkFun_KiCad_CAMmer/assets/601236/a32ba454-27e6-4ade-bd0b-45580f7ca4be)

However, when I run the cam tool, the ordering instructions text file looks like this:

![image](https://github.com/sparkfun/SparkFun_KiCad_CAMmer/assets/601236/ba2b83d9-2659-4b3e-b528-93b5fbc1d1c0)

It seems to be skipping over the "2oz Outer Layer". Or possibly over-writing it when it finds the inner.

Thanks!

**Comment by PaulZC** [comment]
It needs to be something like:

```
if "weight" in line or "Weight" in line or "WEIGHT" in line or "oz" in line or "Oz" in line or "OZ" in line:
    if copperWeight is None:
        copperWeight = line
    else:
        copperWeight = copperWeight + "\n" + line
```

**Comment by PaulZC** [comment]
@lewispg228 : please try v1.1.3 : https://github.com/sparkfun/SparkFun_KiCad_Panelizer/actions/runs/8154474788

You will need to:
* Download the artifact
* Open the KiCad Pugin and Content Manager
* Click the "Installed" tab
* Click the Panelizer "Uninstall" button
* Click "Apply Pending Changes"
* Click "Install from File"
* Select the v1.1.3 zip file
* Close the PCM
* Open a PCB
* Give it a try!

**Comment by PaulZC** [comment]
This is resolved in v1.1.3 and v1.2.0.

v1.2.0 is available now on the [Releases page](https://github.com/sparkfun/SparkFun_KiCad_Panelizer/releases). It will be available through the PCM when the KiCad team have time to merge it.
